Block/Checkout/Onepage/Success/TescoAdditionalInformation.php

```diff
@@ -0,0 +1,53 @@
+<?php
+namespace Omise\Payment\Block\Checkout\Onepage\Success;
+
+class TescoAdditionalInformation extends \Magento\Framework\View\Element\Template
+{
+    /**
+     * @var \Magento\Checkout\Model\Session
+     */
+    protected $_checkoutSession;
+
+    /**
+     * @param \Magento\Framework\View\Element\Template\Context $context
+     * @param \Magento\Checkout\Model\Session $checkoutSession
+     * @param array $data
+     */
+    public function __construct(
+        \Magento\Framework\View\Element\Template\Context $context,
+        \Magento\Checkout\Model\Session $checkoutSession,
+        array $data = []
+    ) {
+        $this->_checkoutSession = $checkoutSession;
+        parent::__construct($context, $data);
+    }
+
+    /**
+     * Return HTML code with tesco lotus payment infromation
+     *
+     * @return string
+     */
+    protected function _toHtml()
```

Comment:
@jacekstan (optional) can we refactor this method to make it return at only one place?
It seems like it can be a little more shorter.